com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,143 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --filesystem=xdg-config/gtk-3.0:ro
+  - --persist=.vscode-oss
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --filesystem=xdg-run/gnupg:ro
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --system-talk-name=org.freedesktop.login1
+separate-locales: false
+add-extensions:
+  com.visualstudio.code.tool:
+    directory: tools
+    subdirectories: true
+    version: '23.08'
+    add-ld-path: lib
+    no-autodownload: true
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - '*.la'
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - shared-modules/libusb/libusb.json
+  - name: codium-insiders
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 codium-insiders.svg /app/share/icons/hicolor/scalable/apps/com.vscodium.codium-insiders.svg
+      - |
+        for size in 16 32 48 64 128 256 512; do
+          rsvg-convert --width ${size} --keep-aspect-ratio --format png --output codium-insiders_${size}.png codium-insiders.svg || exit 1
+          install -Dm644 codium-insiders_${size}.png /app/share/icons/hicolor/${size}x${size}/apps/com.vscodium.codium-insiders.png || exit 1
+        done
+      - install -Dm644 com.vscodium.codium-insiders.metainfo.xml -t /app/share/metainfo
+      - cp /usr/bin/ar /app/bin
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && ln -s /usr/lib/${ARCH_TRIPLE}/libtinfo.so/app/lib/libtinfo.so.5
+      - ar x codium.deb
+      - tar xf data.tar.xz
+        # HACK: Use directory for IPC sockets that is shared between Flatpak instances
+        # Replace XDG_RUNTIME_DIR by XDG_RUNTIME_DIR/app/FLATPAK_ID
+      - find usr/share/codium-insiders -type f -print0 | xargs -0 sed -i "s/\bjoin([^)]*xdg_\?runtime_\?dir[^,)]*/\0,'app',process.env.FLATPAK_ID/gI"
+        # and disable integrity check
+      - sed -e 's/"checksums":/"_checksums":/' -i usr/share/codium-insiders/resources/app/product.json
+      - mv usr/share/codium-insiders /app/share
+      - sed -e "s|/usr/|/app/|g" -e "s/^Icon=.*/Icon=com.vscodium.codium-insiders/" -i usr/share/applications/*.desktop
+      - sed -e "s|share/codium-insiders/codium-insiders|bin/codium-insiders|g" -i usr/share/applications/*.desktop
```

Comment:
Use desktop-file-edit

Reply:
How I can use `desktop-file-edit` to modify `Exec` on `[Desktop Action new-empty-window]` because when I tried `desktop-file-edit --set-key=Exec --set-value="/app/bin/codium-insiders --new-window %F" insiders.desktop` this only apply changes to `[Desktop Entry]` not even apply for both keys

Reply:
Doesn't look like it's possible.